test-utils/testHelpers.js

```diff
@@ -0,0 +1,22 @@
+// this file must only contain js code written in ES3 as it's used in tests
```

Comment:
this ideally would be enforced using eslintrc and "overrides" - is there a reason this *must* be a string? could it be an actual JS function, thus lintable, and when concatenated into a string elsewhere, use Function.prototype.toString?

Reply:
If we don't store it as a string, we're relying on the (proper) implementation of Function.prototype.toString() in the executing environment, whatever that may me.

I cannot judge whether it's safe to rely on that... thoughts?

Reply:
yeah, that's a fair point :-/ probably best to keep it as a string just in case.